meinberlin/apps/maptopicprio/api.py

```diff
@@ -0,0 +1,25 @@
+from django.db.models import Model
+from rest_framework.serializers import ModelSerializer
+
+from meinberlin.apps.maptopicprio.models import MapTopic
+from meinberlin.apps.maptopicprio.serializers import MapTopicSerializer
+from meinberlin.apps.topicprio.api import BaseTopicViewSet
+from meinberlin.apps.topicprio.api import ModuleMixin
+from meinberlin.apps.topicprio.api import PermissionInfoMixin
+from meinberlin.apps.topicprio.api import TopicFilterInfoMixin
+
+
+class BaseMapTopicViewSet(BaseTopicViewSet):
+    model: Model = MapTopic
+    serializer_class: ModelSerializer = MapTopicSerializer
+
+    def get(self, request):
```

Comment:
do we need this? this is new, right?

Reply:
I thought that what we needed to initialize the extended model? but I just got it from the internet

Reply:
don't think so unless I'm missing something.